packages/node_modules/@node-red/nodes/locales/en-US/function/80-template.html

```diff
@@ -47,5 +47,9 @@ <h3>Details</h3>
     <code>{{global.name}}</code>, or for persistable store <code>store</code> use <code>{{flow[store].name}}</code> or
     <code>{{global[store].name}}</code>.
     <p><b>Note: </b>By default, <i>mustache</i> will escape any non-alphanumeric or HTML entities in the values it substitutes.
-       To prevent this, use <code>{{{triple}}}</code> braces.
+       To prevent this, use <code>{{{triple}}}</code> braces.</p>
+    <p>You can change the characters used for templating (if for example you need to use {{ }} within your content) by adding a line to the top of the template itself,
+    to change to use [[ ]] add:
+    <code>{{=[[ ]]=}}</code>
+    To the start of your template.</p>
```

Comment:
```suggestion
       To prevent this, use <code>{{{triple}}}</code> braces.</p>
    <p>If you need to use `<code>{{ }}</code> within your content, you can change the characters
       used to mark the templated sections. For example, to use <code>[[ ]]</code>
       instead, add the following line to the top of the template:</p>
    <pre>{{=[[ ]]=}}</pre>
```

Reply:
A little word-smithing and a bit of formatting.